### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DanielMitiku**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (inhaq) [inline index.html:1]: Good work on adding a `doctype` 😊

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<html>
```
Comment (inhaq) [inline index.html]: Add a `lang` attribute to the `html` tag to declare the language of this document.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<head>
+	<title>Embedding images and videos</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<nav id="navbar">
+	<ul>
+		<li><a href="#"><img  src="menu.PNG"></a></li>
```
Comment (inhaq) [inline index.html]: An `img` element must have an `alt` attribute that is to be rendered when the element to which it is applied cannot be rendered.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+		<li><a href="#"><img src="youtube.PNG"></a></li>
+		<li id="search"><input id="input" type="text" placeholder="Search" width="400px" margin-right="40px"><button id="search-button"><img id="button" src="search.PNG" ></button></li></a></li>
```
Comment (inhaq) [inline index.html]: Attribute `width` and `margin-right` not allowed on element `input`.
No opening tag for `a` exists also `li` tag is closed twice.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	</ul>
+	<hr>
+	</nav>
+
+
+	
+
+	<div id="main_body">
+	<div class="iframe">
+		<iframe width="600" height="400" src="https://www.youtube.com/embed/NnO08HnZf10" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (inhaq) [inline index.html]: The `frameborder` attribute on the `iframe` element is obsolete.

**Review by inhaq — CHANGES_REQUESTED** [review]
### Great work on your first project 🏆
Kindly look into these changes mentioned below. Also It doesn't look fine with me in chrome, check screenshot [here](https://snag.gy/wisSJ2.jpg). 
I would also recommend to add `img` folder and put all images in it and update links accordingly.
In order to see whether your HTML is good, you can validate it [here](https://validator.w3.org). Once you finished with the project, kindly resubmit it and mentioned me(Inzimam) in reviewed before. Happy coding!

File: index.html [hunk 6]
```diff
@@ -1,20 +1,26 @@
...
 	<title>Embedding images and videos</title>
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
 	<nav id="navbar">
 	<ul>
-		<li><a href="#"><img  src="menu.PNG"></a></li>
-		<li><a href="#"><img src="youtube.PNG"></a></li>
-		<li id="search"><input id="input" type="text" placeholder="Search" width="400px" margin-right="40px"><button id="search-button"><img id="button" src="search.PNG" ></button></li></a></li>
+		<li><a href="#"><img  src="img/menu.PNG" alt="Missing Img"></a></li>
```
Comment (inhaq) [inline index.html]: `alt` tag should have text related to the image. For example, if it's a cat image than `alt` should be a cat. It should contain information about that image.

**Review by inhaq — CHANGES_REQUESTED** [review]
Good work on applying all these suggestions but you need to make the design look bit better. Let me [share the screenshot](https://snag.gy/XHkcl3.jpg). Kindly look into it and update your code. Once you're done, kindly resubmit the project.

**Comment by DanielMitiku** [comment]
Hey, It looks fine in our browser.

**Comment by DanielMitiku** [comment]
capture

**Comment by inhaq** [comment]
> Hey, It looks fine in our browser.

I can see it looks fine on 720 resolution screens but try to zoom out and you'll see the problem on your screen as well. Also, one more thing, check inspector tools in chrome and see your body is broken. Let me know if you need more info.

**Comment by DanielMitiku** [comment]
So what do we do! This is our first assignment and it doesn't talk about screen sizes. May be you can help us!

**Comment by inhaq** [comment]
[This](https://www.codeproject.com/Articles/762201/Adjusting-your-websites-to-fit-all-types-of-resolu) might help you, the goal of this project is not to make it responsive for every screen but at least it should work on a few laptop resolutions. If you guys need help, don't hesitate to ask or you can go to your dashboard and click on technical support to help with technical problems.

**Comment by DanielMitiku** [comment]
Did you see the screenshot? It works on our laptop!

**Comment by inhaq** [comment]
> Did you see the screenshot? It works on our laptop!

I can see it like [this](https://snag.gy/cnSyz3.jpg).
1. Doesn't look right, as I mentioned in this screenshot as point 1.
2. Why is this line popping up on screen?

You guys did an awesome work as a first project but I want you to make it as good as possible. If you're happy with what you've done, kindly make these two changes and submit it. I'll 👍🏻. But I'm here to help you guys deliver excellent projects which look good on your CV. Hope you understand. In case of technical questions, you can reach to technical support in your dashboard.

**Comment by DanielMitiku** [comment]
Hey inhaq, I've edited it. Please take a look at it.

**Comment by inhaq** [comment]
> Hey inhaq, I've edited it. Please take a look at it.

I just looked into it. [This](https://snag.gy/BXouvY.jpg) needs some styling. Specifically, views, share, likes should be aligned with the title. 
Take your time with this project, the more time you make this one awesome, the lesser time you'll spend on new projects. If you rushed this one, then you guys will get more problems in next HTML/CSS projects as well. I want you to go through more readings and guides to understand this part of program very well. I'm sure you guys will do the best possible work very soon.

**Comment by DanielMitiku** [comment]
Can you look at it now!

**Comment by inhaq** [comment]
I really think you did a great job for your screen size, but as the project states, this should be responsive to more than just 1 screen size. If you were to put this project in your portfolio right now, and a potential employer was to look at it with different screen size, they wouldn't think you did a very good job. All of the projects you make now will be part of your final portfolio and are supposed to help you get a job. This one won't do that unless the employer happens to have the exact same screen size as you. I'd recommend you reading [here](https://www.quackit.com/css/), [here](https://30-seconds.github.io/30-seconds-of-css/) and [here](https://www.w3schools.com/css/default.asp) to brush up on responsive design(at least for laptop resolutions, no need for media queries) to make sure your project will be good to get the thumbs up.

**Comment by DanielMitiku** [comment]
The here, here and here are not links

**Comment by DanielMitiku** [comment]
Did you look at the changes we made

**Comment by inhaq** [comment]
### 👍🏻 for this project 🥇
Congrats on completing this project. I know you guys have tried way hard and in the mean time also learn lots of things. But one thing which I wanted to say to keep in mind to have more effort in next project. This one is the first one and I know you guys will do next one way better. Go ahead and submit the project completion form. Happy coding 🎉

**Review by inhaq — APPROVED** [review]
### 👍🏻 for this project 🥇
Congrats on completing this project. I know you guys have tried way hard and in the mean time also learn lots of things. But one thing which I wanted to say to keep in mind to have more effort in next project. This one is the first one and I know you guys will do next one way better. Go ahead and submit the project completion form. Happy coding 🎉

**Pull request merged**